homeassistant/components/deconz/__init__.py

```diff
@@ -119,6 +119,13 @@ def async_add_remote(sensors):
 
     deconz.start()
 
+    device_registry = await \
+        hass.helpers.device_registry.async_get_registry()
+    device_registry.async_get_or_create(
```

Comment:
As this example might be copied a lot, I would prefer that we use named arguments to clarify what is what:

```python
device_registry.async_get_or_create(
  identifiers=[[DOMAIN, deconz.config.bridgeid]],
  manufacturer='Dresden Elektronik',
  # etc
)
```